app/views/users/index.html.erb

```diff
@@ -1,17 +1,19 @@
 <main>
-    <section>
-      <ul>
-        <% @users.each do |user|%>
-          <%= link_to user do %>
-            <li class="card hover">
-              <div class="user-image border"></div>
-              <div class="user-data border">
-                <h1><%= user.name %></h1>
-                <p class="number-posts">Number of posts: <%= user.posts_counter %></p>
-              </div>
-            </li>
-          <% end %>
+  <section>
+    <ul>
+      <% @users.each do |user| %>
+        <%= link_to user do %>
+          <li class="card hover">
+            <div class="user-image border">
+              <%= image_tag user.photo.url %> <!-- Assuming the user.photo.url holds the URL of the user's photo -->
+            </div>
```

Comment:
![image](https://github.com/Cisco-the-wayword/BlogApp/assets/93039359/2ac8f6be-80ac-4666-b538-e8582c56c059)

- [ ] Great work so far :+1: but it would be great if you fix the image error which you can see in the image above :arrow_up: Kindly fix this error to make your app work as expected :+1: as suggested by the previous reviewers :blush:

### :green_circle: I have fixed this error by replacing your line 8 with the below line :arrow_down:

```ruby
<img class = "user_image" src=<%= user.photo %> alt="user image" />
```